Lib/netrc.py

```diff
@@ -165,12 +170,7 @@ def _security_check(self, fp, default_netrc, login):
 
     def authenticators(self, host):
         """Return a (user, account, password) tuple for given host."""
-        if host in self.hosts:
-            return self.hosts[host]
-        elif 'default' in self.hosts:
-            return self.hosts['default']
-        else:
-            return None
+        return self.hosts.get(host, self.hosts.get('default'))
```

Comment:
I have a feeling that this might not be relevant to the fix and would be better in a separate refactoring PR.